_run.ts

```diff
@@ -70,22 +69,75 @@ namespace runner {
         showHint(heading, "Available scripts:", await getScriptNames());
     }
 
+    function printJsonSchemaErrors(filename: string, errors: ValidationError[]): void {
+        const errorLines = errors.map(error => `- ${error.property} ${error.message}`);
+        const file = path.relative(__dirname, filename);
+        const s = errorLines.length > 1 ? "s" : "";
+        console.error(
+            `${ANSI_YELLOW_BRIGHT}⚠ ` +
+                `${ANSI_RED}The file ${file} has the following formatting error${s}:` +
+                ANSI_RESET
+        );
+        errorLines.forEach(line => console.error(ANSI_RED + line + ANSI_RESET));
+        console.error();
+    }
+
+    async function getValidatedJson<T>(
+        jsonFilename: string,
+        schemaFilename: string,
+        noSuchFileCB: () => any
```

Comment:
What does CB mean in this context?

Reply:
Answering to myself, probably callback. Maybe use Callback instead of CB so that is more clear for reader?

Reply:
"cb" is such a common abbreviation in TypeScript/JavaScript that I'll let it be this time